src/Pages/ProductListing/FiltersContainer.jsx

```diff
@@ -0,0 +1,252 @@
+            </div>
+
+            {/* <!-- SEPERATOR --> */}
+            <hr className="solid separator"/>
+
+            {/* <!-- PRODUCT TAGS --> */}
+            <p className="text-lg heading">Product tags</p>
+            <div className="col-12 filters-container"> 
+            {/* FUNCTIONALITY YET TO BE IMPLEMENTED                    */}
+                {/* <InputField 
+                    type='text'
+                    title='Product tag'
+                    name='productTag'
+                    placeholder='Enter product tag..'
+                    err={false}
+                    // value={}
+                    //onChange={(e)=> handlebrandSearch(e)}
+                /> */}
+                
+                <div className="col-12 prdct-tag-container">
+                   {prdctTags.length?
+                        prdctTags.map((cVal,cIndx) =>{
+                            return <div key={cIndx} className="col-12 prdct-tag-filter-container">
+                                        <Input className="checkbox" type='checkbox' id={`checkbox-${cIndx}`}
+                                            name={cVal.prdctTagTile}
+                                            value={cVal.prdctTagTile}
+                                            onChange = {()=>handleProductsTagCheckBox(cVal.prdctTagTile)}
+                                            checked={cVal.isSelected}
+                                        />
+                                        <label className="text-xs lbl-prdct-tag">{cVal.prdctTagTile}</label>
+                                        <label className="lbl-count">5</label>
+                                    </div>
+                        })
+                    :null} 
+                </div>
+            </div>
+
+            {/* <!-- SEPERATOR --> */}
+            <hr className="solid separator"/>
+
+            {/* <!-- PRICE FILTER --> */}
+            <p className="text-lg heading">Price</p>
+            <div className="col-12 filters-container">
+                <div className="price-filter">
+                    {/* <input className="radio-btn" type="radio"/> */}
+                    <Input className="radio-btn" type='radio' id="radiobtn-lowToHigh" 
+                            name='lowToHigh' value="lowToHigh" onChange={() => dispatch({ type: "LOW_TO_HIGH" })}
+                            checked={sortBy === "LOW_TO_HIGH"}
+                    />
+                    <label className="lbl-price" for="lowToHigh">Low To High</label>
+                </div>
+
+                <div className="price-filter">
+                    <Input className="radio-btn" type='radio' id="radiobtn-HighTolow" 
+                            name='HighTolow' value="HighTolow" onChange={() => dispatch({ type: "HIGH_TO_LOW" })}
+                            checked={sortBy === "HIGH_TO_LOW"}
+                    />
+                    <label className="lbl-price"  for="HighTolow">High To Low</label>
```

Comment:
Suggestion: use `htmlFor` instead of  `for` in react

Reply:
ok